games-emulation/ppsspp/ppsspp-1.11.3_p20210627.ebuild

```diff
@@ -0,0 +1,121 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake desktop xdg
+
+DESCRIPTION="A PSP emulator written in C++"
+HOMEPAGE="https://www.ppsspp.org/"
+
+MY_COMMIT="ff2a6eeda2d00210f1a1f7bc0ab0ad99e7bc60b6"
```

Comment:
Just a nitpick I'd prefer above description still, that's how most ebuilds do it, and it maintains the common desc/homepage/uri block together. And it's nice to have something that needs attention on bumps to be higher up for editing, people put other version variables there usually.

e.g.
```
inherit cmake desktop xdg

MY_COMMIT=...

DESCRIPTION
HOMEPAGE
SRC_URI